src/api/telegram-subscription/controllers/telegram-subscription.ts

```diff
@@ -25,6 +25,21 @@ export default factories.createCoreController(MODULE_ID, ({strapi}) => {
 
       return true
     },
+    async removeSubscription(context) {
```

Comment:
So this call never fails/returns false?

Reply:
It will throw inside of `checkSubscription` if there is no valid authorization